### ISSUE TITLE
Translation of aria-label

### ISSUE THREAD

**Issue by SventB** [benjaminkott/bootstrap_package#723]
For the attribute "aria-label", you use english text, but the texts should be in the selected language, because this attribute is used for screenreaders.

**Comment by benjaminkott** [comment]
@SventB thx, was not aware of that these were missing, found, one in breadcrumb and another one in the nav-toggle.
